kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/sink/Neo4jService.kt

```diff
@@ -39,7 +40,7 @@ class Neo4jService(private val config: Neo4jSinkConnectorConfig):
     private val transactionConfig: TransactionConfig
 
     init {
-        val configBuilder = Config.builder()
+        val configBuilder = Config.builder().withLogging(Logging.slf4j())
```

Comment:
why this approach and not the same of the other PR?

Reply:
Hi Andrea! Thanks for taking your  time and looking at my PRs! Like I've said in the support ticket that's only because I'm not Java/Kotlin guy and wanted your experienced look at these two approaches. I saw you mentioned you liked the other approach, I can definitely port it to this PR. Also I wasn't really sure which is the best way because [offical neo4j driver API docs](https://neo4j.com/docs/api/java-driver/current/org/neo4j/driver/Logging.html) literally say: _This logging method is the preferred one and relies on the SLF4J implementation available in the classpath or modulepath_

Also it worth mentioning that ideally the connector logs should be managed in the similar manner [the official docs suggests it](https://docs.confluent.io/platform/current/connect/logging.html#using-the-kconnect-log4j-properties-file). Where they could be configured via container env variables picked up by native confluent kafka-connect env tools
```
log4j.logger.org.apache.kafka.connect.runtime.WorkerSinkTask=DEBUG
log4j.logger.org.apache.kafka.connect=DEBUG
```
Please let me know if you still want to port PR form brach 3.5? Thanks again!

Reply:
If you look at the code of the java driver:

https://github.com/neo4j/neo4j-java-driver/blob/4.3.4/driver/src/main/java/org/neo4j/driver/Config.java#L272

It uses the same API that you used in the other PR.

So maybe I misunderstood the goal of this and other PR. Do you want to provide a config option for the log level of the java driver? Or?

Reply:
Thanks for the link! Yeah, I do, I've  ported the PR